rds-postgresql-kfh-transform/main.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"bytes"
+	"compress/gzip"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/honeycombio/agentless-integrations-for-aws/common"
+	"github.com/honeycombio/honeytail/event"
+	"github.com/honeycombio/honeytail/parsers/postgresql"
+)
+
+var parser *postgresql.Parser
+
+func handler(ctx context.Context, input events.KinesisFirehoseEvent) (events.KinesisFirehoseResponse, error) {
```

Comment:
For both handlers: would it be possible to add a test for the handler function?

Reply:
I definitely think the tests will be really valuable to add and have added an [Asana story](https://app.asana.com/0/1198607545155957/1203323496651406/f) to make sure we don't forget to do it before pointing users as the Terraform module that will be consuming this. It would be really great if we could get this merged before adding the tests so we can validate our IaC works to pull this code correctly once the binary is built